docs/120_contribute/README.md

```diff
@@ -60,6 +60,12 @@ Please follow the coding conventions and style used in the project. Use ESLint a
 
 ### Run Tests Locally
 
+> ⚠️ **Warning: Be cautious when creating test cases in `journey/`!**
+>
+> - Test cases that capture end-to-end/journey behavior are usually stored in [pepr-excellent-examples](https://github.com/defenseunicorns/pepr-excellent-examples) or run as a Github workflow (`.github/workflows`).
+> - Journey tests established in `journey/` are from an earlier time in project history.
+> - Issue [pepr/#1140](https://github.com/defenseunicorns/pepr/issues/1140) exists to refactor the test suite under `journey/`.
```

Comment:
```suggestion
```

Not sure this belongs in a contributing file since there have been no community members/users to mention this issue and considering that there is no problem running test locally. We have several issues around improving and changing our test suite which also are not mentioned here, this does not need special call out